app/src/main/java/com/willowtree/vocable/settings/EditCategoryMenuFragment.kt

```diff
@@ -26,16 +25,11 @@ class EditCategoryMenuFragment : BaseFragment<FragmentEditCategoryMenuBinding>()
     override val bindingInflater: BindingInflater<FragmentEditCategoryMenuBinding> =
         FragmentEditCategoryMenuBinding::inflate
 
-    private lateinit var editCategoryMenuViewModel: EditCategoryMenuViewModel
+    private val editCategoryMenuViewModel: EditCategoryMenuViewModel by viewModels()
```

Comment:
ok, so if I understand this correctly, we want to defer the ViewModelStoreOwner to the Activity is some places so that it is longer lived that just the fragment. Is that accurate?

Reply:
Done in 3a709c7

Reply:
> ok, so if I understand this correctly, we want to defer the ViewModelStoreOwner to the Activity is some places so that it is longer lived that just the fragment. Is that accurate?

That is accurate. It's something I'd like to get away from at some point, but it's necessary to maintain existing functionality.